index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <!--NAV VAR-->
+    <header>
+        <nav>
+            <p>My logo</p>
+            <button class="menu"><img src="Img/Icon - Menu.png" alt=""></button>
+                <section class="menudesk">
+                    <a href="">Portfolio</a>
+                    <a href="">About</a>
+                    <a href="">Contact</a>
+                </section>
+        </nav>
+    </header>
+    <br>
+    <section class="info">
+        <div class="desc">
+            <h1>I'm Andrea.<br>Glad to see you!</h1>
+            <h2>I'm Andrea.<br>Glad to see you!</h2>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="enlaces">
+                <p>Let's conect</p>
+                <a href=""><img src="Img/twitter.svg" alt=""></a>
+                <a href=""><img src="Img/Linkedin icon.svg" alt=""></a>
+                <a href=""><img src="Img/medium.svg" alt=""></a>
+                <a href=""><img src="Img/github.svg" alt=""></a>
+                <a href=""><img src="Img/angellist.svg" alt=""></a>
+            </div>
+        </div>
```

Comment:
- [ ] Kindly use an indentation of two spaces in all your documents. THis is beast practice and also a convention.

Reply:
Sorry @Kamba56 but I don't understand this part please, could you give me some example to see how I has to do that?

Reply:
> Sorry @Kamba56 but I don't understand this part please, could you give me some example to see how I has to do that?

Currently, your tab size is four spaces. You should reduce it to two spaces so that your code will look this.
```html
<div>
  <h1></h1>
  <ul>
    <li>
      <a></a>
    </li>
  </ul>
</div>
```

Notice how the indentation is just two spaces and also less than yours.